required/connection.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
rename from `connection.json` to `connection.txt` and change content to 1 character, literally, 1, should do it.